### DISCUSSION TITLE
Unable to login

### DISCUSSION THREAD

**Discussion by sffahad08** [kevinzg/facebook-scraper#511] [General]
Please help me

**Comment by neon-ninja** [comment]
What error do you get?

**Comment by sffahad08** [comment]
I am a Student and I don't understand what is github. But all I understand
github is for programmers. So if you are
.
Please help me I failed to login my account because I lost google
authenticator for 2 step verification.
9168901305
Password-fahadmp2022

**Reply by neon-ninja** [reply]
That's not something I can help you with. And you shouldn't post your password publicly. See https://m.facebook.com/help/147926301947841

**Comment by sffahad08** [comment]
But i can't help myself I'm helpless. I tried so many times, please help
me.i think you can do it for me

**Reply by neon-ninja** [reply]
No

**Comment by sffahad08** [comment]
Nick, I can tell you one thing, when you'll wake on 25october, you will
wake up with an innovative idea that will change you if you help me
